backend/src/users/status/status.gateway.ts

```diff
@@ -6,9 +6,9 @@ import {
 } from '@nestjs/websockets';
 import { Server, Socket } from 'socket.io';
 import { SessionsService } from '../sessions/sessions.service';
-import { UsersService } from '../users/users.service';
-import { Session } from '../core/entities';
-import { UserStatus } from '../core/entities';
+import { UsersService } from '../users.service';
+import { Session } from '../../core/entities';
+import { UserStatus } from '../../core/entities';
```

Comment:
```suggestion
import { Session, UserStatus } from '../../core/entities';
```
vem do mesmo local, dá pra por no mesmo `import`